llama_stack/providers/remote/inference/together/together.py

```diff
@@ -175,6 +179,14 @@ def _build_options(
             else:
                 raise ValueError(f"Unknown response format {fmt.type}")
 
+        if logprobs and logprobs.top_k:
+            if logprobs.top_k != 1:
```

Comment:
I think we should raise a `ValueError` here. Subtly altering a request is not right.